### PR TITLE
Add used imports documentation/tutorial.md

### PR DESCRIPTION
**Pull request by lopezac**
* Add used imports combine, createEvent and createStore from effector.
* Add final semicolon to export.
* Add changes to english and russian file.

### CODE REVIEW

**Comment by illright** [comment]
Can't approve this, idk if this is needed

**Comment by illright** [comment]
Why did you close it? I figured @unordinarity might be able to approve this. It's just that I haven't worked with Effector before

**Review by unordinarity — APPROVED** [review]

**Comment by unordinarity** [comment]
@lopezac thank you for those improvements, please merge when you ready

**Pull request merged**
